node/src/db_config/config_dao.rs

```diff
@@ -212,25 +227,22 @@ mod tests {
     #[test]
     fn set_and_get_work() {
         let home_dir = ensure_node_home_directory_exists("config_dao", "set_and_get_work");
-        let subject = ConfigDaoReal::new(
-            DbInitializerReal::default()
-                .initialize(&home_dir, DbInitializationConfig::test_default())
-                .unwrap(),
-        );
+        let subject = make_subject(&home_dir);
         let modified_value = ConfigDaoRecord::new(
             "consuming_wallet_private_key",
             Some("Two wrongs don't make a right, but two Wrights make an airplane"),
             true,
         );
+
         subject
             .set(
                 "consuming_wallet_private_key",
                 Some("Two wrongs don't make a right, but two Wrights make an airplane".to_string()),
             )
             .unwrap();
-
```

Comment:
We need a blank line here, to separate the act, maybe you removed it by mistake, also you can eliminate the blank line above the assert statements.